packages/devtools/lib/src/logging/logging.dart

```diff
@@ -172,7 +172,9 @@ class LoggingScreen extends Screen {
     service.onGCEvent.listen(_handleGCEvent);
 
     // Log `dart:developer` `log` events.
+    // TODO(devoncarew): Remove `_Logging` support on or after approx. 10/1/2019.
```

Comment:
Nit: Unambiguous date formats may be  better for things that could be around a while.

Reply:
👍 done